stix_shifter_modules/cybereason/configuration/lang_en.json

```diff
@@ -3,7 +3,7 @@
         "host": {
             "label": "Management IP address or Hostname",
             "placeholder": "192.168.1.10",
-            "description": "Specify the IP address or  hostname of the data source so that IBM Cloud Pak for Security can communicate with it"
+            "description": "Specify the IP address or hostname of the data source"
```

Comment:
may be revert back this change?

Reply:
I thought this should not be cp4s specific
I can revert it back

Reply:
no that was not me
https://github.com/opencybersecurityalliance/stix-shifter/commit/b8f5f22984270dfc3862526cc96148e4fc8b9209
done by thangaraj-ramesh